src/Components/Aspire.MongoDB.Driver/Aspire.MongoDB.Driver.csproj

```diff
@@ -20,14 +21,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="AspNetCore.HealthChecks.MongoDb" />
-    <PackageReference Include="MongoDB.Driver" />
+    <PackageReference Include="AspNetCore.HealthChecks.MongoDb" VersionOverride="[8.1.0,9.0.0)" />
```

Comment:
It's there because Directory.Packages (and/or Versions.props) doesn't handle ranges?

Reply:
No. This is necessary because Directory.Packages.props defines the `9.0.0` version, but this package needs the previous version - 8.1.0 since it uses MongoDB v2.